### ISSUE TITLE
Add lightbox to acronyms in text

### ISSUE THREAD

**Issue by ybznek** [OpenSCAP/website#8]
It could help users to read text without switching betwen text/acronym page.

**Comment by ybznek** [comment]
Maybe we can generate some javascript code containing all acronyms.

So, every page can link the script and run it (JS can process elements e.g. with defined class, find the acronyms in the text and add lightbox to them).

Advantage:
+ script with acronyms can be downloaded only once, browser will cache it
+ We don't need to use some Wordpress plugin to preprocess our texts 

Disadvantage:
+ maybe some performance issue with browsing from old mobile devices etc.
+ acronyms need to be processed every time when user open page

**Comment by lhorakov** [comment]
At this page we have implemented (thanks to you) the lightbox for acronym "OVAL": http://openscap-lhorakov.rhcloud.com/features/vulnerability-assessment/

It can be added  into each page with acronyms. Please contact me for more detailed information how to do it.

**Comment by msrubar** [comment]
It would be great if the lightbox would disappear when a user clicks somewhere else then on the lightbox. Take a look [here](http://highslide.com/) and click on the picture of the house (it's possible to close the preview by clicking on the picture or somewhere else).

**Comment by isimluk** [comment]
Ok, so this ticket is for lightbox's javascript. Let's track all the lightbox occurrences in #27

**Comment by isimluk** [comment]
Tentatively to 2.0 milestone. This is nice to have, but I feel we can safely GA without this.

**Comment by isimluk** [comment]
I think this can be closed now, right?
